sdk-utils/src/main/kotlin/com/rakuten/tech/mobile/sdkutils/eventlogger/EventLogger.kt

```diff
@@ -158,17 +161,91 @@ object EventLogger {
         }
     }
 
+    @SuppressWarnings(
+        "LongParameterList"
+    )
+    private fun logEvent(
+        eventType: EventType,
+        sourceName: String,
+        sourceVersion: String,
+        errorCode: String,
+        errorMessage: String,
+        info: Map<String, String>?
+    ) {
+        if (!isConfigureCalled ||
+            !eventLoggerHelper.isEventValid(sourceName, sourceVersion, errorCode, errorMessage)) {
+            log.warn("Event Logger is not configured or event contains an empty parameter, skip")
+            return
+        }
+
+        tasksQueue.safeExecute {
```

Comment:
Looking at its name -> variable `tasksQueue` seems to be a queue i.e. which may have multiple items in it.
But it is an instance of ExecutorService =>  `tasksQueue = Executors.newSingleThreadExecutor()`

Can we rename it  ?

Reply:
Yes, for brevity renamed it to `executorService`